src/applications/static-pages/createAdditionalInfoWidget.js

```diff
@@ -34,7 +34,7 @@ export default function createAdditionalInfoWidget() {
       button.addEventListener('click', () => {
         const ariaExpanded = JSON.parse(button.getAttribute('aria-expanded'));
 
-        button.setAttribute('aria-expanded', !ariaExpanded);
+        button.setAttribute('aria-expanded', `${!ariaExpanded}`);
```

Comment:
ugh, it JS doesn't automatically treat it as a string :( Well I was hoping for something cleaner, but this hasn't gone like I planned.